README.md

```diff
@@ -393,5 +393,20 @@ docs/         # Directory for static pages to be included in the documentation.
     ...       # Other markdown pages, images and other files.
 autora/          # The directory containing the source code.
 ```
+# Release Process
 
+## PyPi
 
+We use the standard poetry publishing workflow as outlined here: https://python-poetry.org/docs/libraries/#publishing-to-pypi
+
+## Conda
+
+This directory includes files for packaging the code for `conda`.
+- [`meta.yaml`](meta.yaml) is the `conda` recipe (configuration file)
+- [`publish-on-anaconda-org.sh`](publish-on-anaconda-org.
+sh) is a script which runs the packaging and outputs the package into the `./dist` directory, then uploads the files to [anaconda.org](https://anaconda.org).
+
+To create and publish the `conda` package:
+- Update `./meta.yaml` with any changed dependencies, to match `pyproject.toml` and taking into account the dependencies which are actually available on the anaconda channels we use: defaults and pytorch. Commit these changes.
+- 🐛 Bugfix: While poetry >=1.2 is not available on anaconda.org, delete the [tool.poetry.group...] parts of pyproject.toml. These are recognized as incorrect Poetry configuration in poetry 1.1 and below, and will cause the build to fail. Don't commit these changes.  
```

Comment:
```suggestion
- 🐛 Bugfix: Until poetry >=1.2 is available on anaconda.org (you should check each time we publish until it is), delete the [tool.poetry.group...] parts of pyproject.toml. These are recognized as incorrect Poetry configuration in poetry 1.1 and below, and will cause the build to fail. Don't commit these changes.  
```

Reply:
is this phrasing (and nudge) more clear or confusing to you?

Reply:
Sounds good to me!